felix/fv/xdp_test.go

```diff
@@ -395,6 +398,7 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ XDP tests with initialized
 			It("should block ICMP too", func() {
 				client, server := clientServerIndexes("tcp")
 
+				time.Sleep(applyPeriod)
```

Comment:
Do we need a sleep here?

Reply:
It won't work without it. The policy program needs some time to get applied.